### PR TITLE
Add Mirantis.Lens 5.1.0

### PR DESCRIPTION
**Pull request by piraces**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21086)

### CODE REVIEW

File: manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Mirantis.Lens
+PackageVersion: 5.1.0
```
Comment (ItzLevvie) [inline manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.1.0-latest.20210715.1
```

Upgrade loop if the version doesn't match the one in Control Panel.

File: manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Mirantis.Lens
+PackageVersion: 5.1.0
```
Comment (ItzLevvie) [inline manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.yaml:5]:
```suggestion
PackageVersion: 5.1.0-latest.20210715.1
```

Upgrade loop if the version doesn't match the one in Control Panel.

File: manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Mirantis.Lens
+PackageVersion: 5.1.0
```
Comment (ItzLevvie) [inline manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.installer.yaml:5]:
```suggestion
PackageVersion: 5.1.0-latest.20210715.1
```

Upgrade loop if the version doesn't match the one in Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
You'll have to update the version folder using GitHub Desktop as well.

File: manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.yaml (hunk 2 above)
Comment (piraces) [reply manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.yaml:5]: Thanks @ItzLevvie for the suggestion! I have already applied it. I will be looking at this for now on.

**Comment by piraces** [comment]
> You'll have to update the version folder using GitHub Desktop as well.

I have now updated the folder and the file `Mirantis.Lens.yaml` to set the same version in all .yml files.
Hope this is correct now. If not please let me know @ItzLevvie . Thank you for all 👍

**Review by OfficialEsco — APPROVED** [review]
Huh, it appears they changed the version scheme in 5.0.2 for some reason..

**Pull request merged**